src/server/socketServer/html.ts

```diff
@@ -13,6 +14,9 @@ export default (base: string, title: string) => (
     <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
     <title>${title}</title>
     <link rel="stylesheet" href="${resourcePath}public/index.css" />
+    <script>
+      window.wettyConfig = ${JSON.stringify(clientConf)};
```

Comment:
Configuration of the client should be done using localstorage https://github.com/butlerx/wetty/blob/master/src/client/options.ts